commons/helper/window_helper.go

```diff
@@ -11,7 +12,10 @@ import (
 var cacheLoaded bool
 
 func AddHertz(txn *badger.Txn, cache *cache.Cache, hertz uint64) *types.Window {
-	epoch := time.Unix(0, types.DispatchEpoch)
+	foo := types.GetConfig().RateLimits
+	fmt.Println(foo)
```

Comment:
Please remove debug lines before merge

Reply:
Ouch, that's embarrassing @davedispatch.